src/Api/Models/Response/SecretResponseModel.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Bit.Core.Entities;
+using Bit.Core.Models.Api;
+
+namespace Bit.Api.Models.Response
+{
+    public class SecretResponseModel : ResponseModel
+    {
+        public SecretResponseModel(Secret secret, string obj = "secret")
+            : base(obj)
+        {
+            if (secret == null)
+            {
+                throw new ArgumentNullException(nameof(secret));
+            }
+
+            Id = secret.Id.ToString();
+            OrganizationId = secret.OrganizationId.ToString();
+            Key = secret.Key;
+            Value = secret.Value;
+            Note = secret.Note;
+            CreationDate = secret.CreationDate;
+            RevisionDate = secret.RevisionDate;
+        }
+
+        public string Id { get; set; }
+
+        public string OrganizationId { get; set; }
+
+        public string Key { get; set; }
+
+        public string Value { get; set; }
```

Comment:
Let's not include Value here. That way they need to do a separate request and we can log that request.

Reply:
resolved in https://github.com/bitwarden/server/pull/2163/commits/1373b497db321fb4dd103995485c222650257138